### DISCUSSION TITLE
No new packs

### DISCUSSION THREAD

**Discussion by Chepycou** [signalstickers/signalstickers#941] [Q&A] [answered]
Hey there, it looks like the latest packs are from more than a month ago, and I submitted one or two packs a few weeks ago and am not seeing them on the website. Is the delay normal (I would completely understand that it takes some time to check the stickers :grin: ), or is it a bug ?

**Comment by romainricard** [comment, marked as answer]
Hi! No bug here, all packs are reviewed before being published. I've been busy lately, and the other reviewers are volunteers. Sorry for the delay!

**Reply by Chepycou** [reply]
Okay thank you !
If there's need for help reviewing don't hesitate to ask ;)
